unlock-app/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@enzoferey/ethers-error-parser": "0.2.3",
```

Comment:
Did you find this library to be good/useful?

Reply:
It does parse some of the common errors better so somewhat useful but nothing with custom errors or wallet errors. Still searching for an overall good way to parse them into something more human readable without resorting to generic messages.